mini-e2e.groovy

```diff
@@ -94,11 +95,23 @@ node('cico-workspace') {
 			sh 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ./prepare.sh ./single-node-k8s.sh root@${CICO_NODE}:'
 			ssh "./prepare.sh --workdir=/opt/build/go/src/github.com/ceph/ceph-csi --gitrepo=${git_repo} --ref=${ref}"
 		}
+		stage('pull base container images') {
+			def base_image = ssh 'source /opt/build/go/src/github.com/ceph/ceph-csi/build.env && echo ${BASE_IMAGE}'
+
+			withCredentials([usernamePassword(credentialsId: 'container-registry-auth', usernameVariable: 'CREDS_USER', passwordVariable: 'CREDS_PASSWD')]) {
+				ssh "podman login --user=${CREDS_USER} --password='${CREDS_PASSWD}'"
+			}
+
+			// base_image is like ceph/ceph:v15
+			ssh "podman pull ${ci_registry}/${base_image} && podman tag ${ci_registry}/${base_image} ${base_image}"
+			// cephcsi:devel is used with 'make containerized-build'
+			ssh "podman pull ${ci_registry}/cephcsi:devel"
+		}
 		stage('build artifacts') {
 			// build container image
 			ssh 'cd /opt/build/go/src/github.com/ceph/ceph-csi && make image-cephcsi GOARCH=amd64 CONTAINER_CMD=podman'
```

Comment:
This will also try to pull the image. can we set the `BASE_IMAGE` in build.env so that it will be used as a base image?

Reply:
the image is already pulled, so it should not pull it again